drizzlepac/runmultihap.py

```diff
@@ -54,6 +54,11 @@ def perform(input_filename, **kwargs):
     debug : Boolean
         display all tracebacks, and debug information?
 
+    input_custom_pars_file : str, optional
+        Represents a fully specified input filename of a configuration JSON file which has been
+        customized for specialized processing. This file should contain ALL the input parameters necessary
+        for processing. If not specified, default configuration parameter values will be used.
+
```

Comment:
Good catch to notice support for custom parameter files was missing from MVM processing.